backend/app/api/routes/scripts.py

```
@@ -165,180 +91,139 @@ async def get_node_content(
-        raise HTTPException(status_code=500, detail=f"Error updating node content: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"Error updating node: {str(e)}")
 
 @scripts_router.post("/insert-node/{script_id}", response_model=Dict[str, Any])
 async def insert_node(
-    script_id: str, 
-    insertion_line: int, 
-    content: str = Body(..., embed=True),
-    node_type: str = Body(..., embed=True),
+    script_id: str,
+    insertion_line: int,
+    node_type: str = Body(...),
```

Ruff (0.14.14)

[warning] Unused function argument: `node_type`

(ARG001)